contrib/src/test/java/com/datatorrent/contrib/hbase/HBaseTransactionalPutOperatorTest.java

```diff
@@ -92,6 +92,12 @@ public int getWindowsFromCheckpoint()
         {
           return 0;
         }
+
```

Comment:
Should be replaced with MockContext
https://github.com/apache/apex-malhar/pull/577/files#diff-ebae1f3ab77d8bc7c7c38d21abf4db32L22